__tests__/queries.test.js

```diff
@@ -154,6 +177,92 @@ describe("Query Tests", () => {
           assert.equal(response.characteristics["Quality"].value, (4 + 3) / 2);
         });
     });
+
+    after((done) => {
+      Review.deleteMany({})
+        .then(() => CharacteristicReview.deleteMany({}))
+        .then(() => Characteristic.deleteMany({}))
+        .then(() => done());
+    });
+  });
+
+  describe('postReview', () => {
```

Comment:
You can use a before() hook if you are manipulating the same data and don't want to re-initialize before each test.

Reply:
I'm not sure if I need to isolate them as I am, but I figure starting from the same entry point every time is safe? I dunno, I guess I'll find out

Reply:
I'm not sure if I need to isolate them as I am, but I figure starting from the same entry point every time is safe? I dunno, I guess I'll find out